### ISSUE TITLE
Event service responds with 500

### ISSUE THREAD

**Issue by pbochynski** [kyma-project/kyma#5933]
**Description**
The event service fails 500 error.

**Expected result**

It works (200 ok).

**Actual result**
```
{
    "status": 500,
    "type": "internal_server_error",
    "message": "Some unexpected internal error occurred, please contact support."
}
```
**Steps to reproduce**

@Abd4llA - got link to the environment when it happens.

The request:
```
curl -X POST \
  https://hostname/appname/v1/events \
  -H 'Accept: */*' \
  -H 'Content-Type: application/json' \
  -H 'Host: event-service.teched-caa384-17.cluster.extend.cx.cloud.sap' \
  -d '{"event-type": "Ticket.Root.Created","event-type-version": "v1","event-id": "00163e88-3bbb-1ee9-bab6-20dfdacc09b2","event-time": "2019-10-08T09:50:11Z","data": {"root-entity-id":"00163E883BBB1EE9BAB62074F11029B1","entity-id":"00163E883BBB1EE9BAB62074F11029B1"}}'
```

**Troubleshooting**

The log output from event publish service:
```
2019/10/08 14:21:49 started knative publish application
2019/10/08 14:21:49 --------------------------------------------------
2019/10/08 14:21:49  port 8080
2019/10/08 14:21:49  max_requests 200
2019/10/08 14:21:49  max_request_size 65536
2019/10/08 14:21:49  max_channel_name_length 33
2019/10/08 14:21:49  trace_debug false
2019/10/08 14:21:49  trace_api_url http://zipkin.kyma-system:9411/api/v1/spans
2019/10/08 14:21:49  trace_host_port 0.0.0.0:0
2019/10/08 14:21:49  trace_service_name event-publish-service
2019/10/08 14:21:49  trace_operation_name publish-the-event
2019/10/08 14:21:49  max_source_id_length 253
2019/10/08 14:21:49  max_event_type_length 253
2019/10/08 14:21:49  max_event_type_version_length 4
2019/10/08 14:21:49  monitoring_port 9090
2019/10/08 14:21:49 --------------------------------------------------
2019/10/08 14:21:49 HTTP metrics server starting on 9090
2019/10/08 14:21:54 ERROR: SendMessage(): 500 Internal Server Error
2019/10/08 14:21:54 failed to send message to the knative channel 'ktfcjd25or2utj5vg2c6zplwcvvja6bdc' in namespace 'kyma-system'
2019/10/08 14:21:55 ERROR: SendMessage(): 500 Internal Server Error
2019/10/08 14:21:55 failed to send message to the knative channel 'ktfcjd25or2utj5vg2c6zplwcvvja6bdc' in namespace 'kyma-system'
2019/10/08 14:21:56 ERROR: SendMessage(): 500 Internal Server Error
2019/10/08 14:21:56 failed to send message to the knative channel 'ktfcjd25or2utj5vg2c6zplwcvvja6bdc' in namespace 'kyma-system'
```

**Comment by pbochynski** [comment]
Restarting natss-dispatcher helps.

**Comment by sayanh** [comment]
This can be due to https://github.com/kyma-project/kyma/issues/3899.
Logs from `natss-dispatcher` can be really helpful here.

**Comment by Abd4llA** [comment]
So far it seems it was fixed by #5196
